src/views/portfolio/licenses/License.vue

```diff
@@ -56,6 +56,7 @@
         <div class="headerText formattedLicenseContent">{{ license.standardLicenseHeader }}</div>
       </b-tab>
     </b-tabs>
+    <b-button v-if="license.isCustomLicense" variant="outline-danger" @click="removeCustomLicense" v-permission="PERMISSIONS.SYSTEM_CONFIGURATION">{{ $t('message.delete_custom_license') }}</b-button>
```

Comment:
I think this should just say "Delete", and the "Custom License" part should be a checkbox like the *OSI Approved* and *FSF Libre* properties above.